wp-content/themes/redmond/web/scss/layout/_footer.scss

```diff
@@ -1,3 +1,56 @@
 .footer {
-
+  background-color: $secondary-color;
+  padding: rem-calc(36px) rem-calc(80px);
+  color: $primary-light-color;
+  h2 {
+    color: $primary-light-color;
+    font-family: 'Lato', sans-serif;
```

Comment:
use the variable instead of hard coding in lato. Search for lato in settings.scss and you should be able to find the variable for it. I think its something like `$body-font-familiy`.